test/gtest/common/test_helpers.cc

```diff
@@ -469,6 +469,26 @@ bool is_interface_usable(struct ifaddrs *ifa)
            !netif_has_sysfs_file(ifa->ifa_name, "wireless");
 }
 
+
+int get_proc_self_status_field(const std::string &parameter)
+{
+    const std::string path("/proc/self/status");
+    std::ifstream proc_stats(path);
+    std::string line, name;
+    int value;
+
+    while (std::getline(proc_stats, line)) {
+        if (!(std::istringstream(line) >> name >> value)) {
+            continue;
+        }
+        if (name.find(parameter) != std::string::npos) {
+            return value;
+        }
+    }
+    UCS_TEST_MESSAGE << path << " does not contain " << parameter << " value";
+    return -1;
```

Comment:
maybe 0 is a better default value?
or pass default value as input param to this function?

Reply:
As I understand `0` is the valid value for this function and I use `-1` as a signal that the value wasn't found at all. I can return `0` and print some message in that case if you prefer so.